asab/web/authn/oauth/middleware.py

```diff
@@ -9,6 +9,41 @@
 #
 
 
+identity_cache = {}
+methods_dict = {}
+g_identity_cache_longevity = 60
+
+
+async def oauthclient_refresh_identity_cache(message_type):
```

Comment:
What is this?
It seems to be an class to me ... including the global variables above.